internal/rbd/replicationcontrollerserver.go

```diff
@@ -123,6 +126,8 @@ func getMirroringMode(ctx context.Context, parameters map[string]string) (librbd
 	switch imageMirroringMode(val) {
 	case imageMirrorModeSnapshot:
 		mirroringMode = librbd.ImageMirrorModeSnapshot
+	case imageMirrorModeJournal:
+		mirroringMode = librbd.ImageMirrorModeJournal
```

Comment:
This should probably be added to the unit-test for the function too, assuming there is a unit-test for it.

The same counts for `validateSchedulingDetails()`.

Reply:
Added the changes to validateSchedulingDetails, unit test for getMirroringMode is missing!